app/views/layouts/_navigation.html.erb

```diff
@@ -9,7 +9,15 @@
   </div>
   <div class="top-bar-right">
     <ul class="menu">
-      <li><a href="#">Login</a></li>
+      <% if user_signed_in? %>
+        <li>
+          <%= link_to "Log out", destroy_user_session_path, method: :delete %>
+        </li>
+      <% else %>
+        <li>
+          <%= link_to "Login", new_user_session_path %>
+        </li>
+      <% end %>
       <li><a href="#">Profile</a></li>
     </ul>
```

Comment:
- [ ] The Log out doesn't work as expected or destroy the user session. Please fix it.

![image](https://user-images.githubusercontent.com/55635977/156013051-635401a3-eb29-4b63-8de2-0f98ebe1504f.png)

Reply:
Thank you for your feedback. Fixed it.